miio/yeelight_dual_switch.py

```diff
@@ -0,0 +1,254 @@
+        {
+            'id': 1,
+            'result': [
+                {'did': 'switch_1_state', 'siid': 2, 'piid': 1, 'code': 0, 'value': False},
+                {'did': 'switch_1_default_state', 'siid': 2, 'piid': 2, 'code': 0, 'value': True},
+                {'did': 'switch_1_off_delay', 'siid': 2, 'piid': 3, 'code': 0, 'value': 300},
+                {'did': 'switch_2_state', 'siid': 3, 'piid': 1, 'code': 0, 'value': False},
+                {'did': 'switch_2_default_state', 'siid': 3, 'piid': 2, 'code': 0, 'value': False},
+                {'did': 'switch_2_off_delay', 'siid': 3, 'piid': 3, 'code': 0, 'value': 0},
+                {'did': 'interlock', 'siid': 4, 'piid': 1, 'code': 0, 'value': False},
+                {'did': 'flex_mode', 'siid': 4, 'piid': 2, 'code': 0, 'value': True},
+                {'did': 'rc_list', 'siid': 4, 'piid': 2, 'code': 0, 'value': '[{"mac":"9db0eb4124f8","evtid":4097,"pid":339,"beaconkey":"3691bc0679eef9596bb63abf"}]'},
+            ]
+        }
+        """
+        print(data)
+        self.data = data
+
+    @property
+    def switch_1_state(self) -> bool:
+        return bool(self.data["switch_1_state"])
+
+    @property
+    def switch_1_default_state(self) -> bool:
+        return bool(self.data["switch_1_default_state"])
+
+    @property
+    def switch_1_off_delay(self) -> int:
+        return self.data["switch_1_off_delay"]
+
+    @property
+    def switch_2_state(self) -> bool:
+        return bool(self.data["switch_2_state"])
+
+    @property
+    def switch_2_default_state(self) -> bool:
+        return bool(self.data["switch_2_default_state"])
+
+    @property
+    def switch_2_off_delay(self) -> int:
+        return self.data["switch_2_off_delay"]
+
+    @property
+    def interlock(self) -> bool:
+        return bool(self.data["interlock"])
+
+    @property
+    def flex_mode(self) -> int:
+        return self.data["flex_mode"]
+
+    @property
+    def rc_list(self) -> str:
+        return self.data["rc_list"]
+
+    def __repr__(self) -> str:
+        s = (
+            "<YeelightDualControlModuleStatus "
+            "switch_1_state=%s, "
+            "switch_1_default_state=%s, "
+            "switch_1_off_delay=%s, "
+            "switch_2_state=%s, "
+            "switch_2_default_state=%s, "
+            "switch_2_off_delay=%s, "
+            "interlock=%s, "
+            "flex_mode=%s, "
+            "rc_list=%s>"
+            % (
+                self.switch_1_state,
+                self.switch_1_default_state,
+                self.switch_1_off_delay,
+                self.switch_2_state,
+                self.switch_2_default_state,
+                self.switch_2_off_delay,
+                self.interlock,
+                self.flex_mode,
+                self.rc_list,
+            )
+        )
+        return s
+
+
+class YeelightDualControlModule(MiotDevice):
+    """Main class representing the air humidifier which uses MIoT protocol."""
```

Comment:
Please update the docstring, and include the model (yeelink.switch.sw1) in it, too.